x-pack/plugins/lists/public/exceptions/components/builder/index.tsx

```diff
@@ -5,6 +5,26 @@
  * 2.0.
  */
 
-export { BuilderEntryItem } from './entry_renderer';
-export { BuilderExceptionListItemComponent } from './exception_item_renderer';
-export { ExceptionBuilderComponent, OnChangeProps } from './exception_items_renderer';
+import { EuiLoadingSpinner } from '@elastic/eui';
+import React, { Suspense, lazy } from 'react';
+
+import type { ExceptionBuilderProps } from './exception_items_renderer';
+export type { OnChangeProps } from './exception_items_renderer';
+
+interface ExtraProps {
+  dataTestSubj: string;
+  idAria: string;
+}
+
+const ExceptionBuilderComponentLazy = lazy(() => import('./exception_items_renderer'));
```

Comment:
What do you think of adding a note here for context about why not to export the component as well? From our zoom it sounded like we wanted to discourage people from using the non-lazy component correct?

Reply:
Yeah, I am fine with adding that note here for sure. I will do that and push up.

Reply:
Added in commit: https://github.com/elastic/kibana/pull/99989/commits/57719a331bdd15e5c98b4d76ca21003474989b99